pkg/pilot/cassandra/v3/options.go

```diff
@@ -85,6 +97,11 @@ func (o *PilotOptions) Complete() error {
 	}
 	o.sharedInformerFactory = informers.NewFilteredSharedInformerFactory(o.navigatorClientset, o.ResyncPeriod, o.GenericPilotOptions.PilotNamespace, nil)
 
+	u, err := url.Parse(o.JolokiaURL)
+	if err != nil {
+		return err
+	}
```

Comment:
What happens if JolokiaURL is not set?

Reply:
I'd like to be able to make it a required flag, but the flags package doesn't support that. 
I added a default.

Reply:
👍 yep, flags package in golang doesn't support required as there is a mantra of "no flag is required", and that defaults should be used instead.

I'm not sure I am convinced by the mantra, but in this case, the use of a default works well. I think it's rare that a user would ever want to override this, given the Pilot itself is what configures Jolokia anyway so it is better informed on the URL to use.